spec/models/user_spec.rb

```diff
@@ -71,6 +75,116 @@
     end
   end
 
+  describe "#required_credits_earned" do
+    before { user.save }
+    let!(:course1) { create :course, degree_requirement: :required_course }
+    let!(:course2) { create :course, degree_requirement: :required_course }
+    let!(:course3) { create :course, degree_requirement: :required_course }
+    let!(:course4) { create :course,
+                     degree_requirement: :distribution_requirement }
+    let!(:course5) { create :course,
+                     degree_requirement: :distribution_requirement }
+    let!(:course6) { create :course, degree_requirement: :free_elective }
+    let!(:taken_course1) { create :taken_course, user: user, course: course1,
+                           quarter: 201415 }
+    let!(:taken_course2) { create :taken_course, user: user, course: course2,
+                           quarter: 201425 }
+    let!(:taken_course3) { create :taken_course, user: user, course: course3,
+                           quarter: 201415 }
+    let!(:taken_course4) { create :taken_course, user: user, course: course4,
+                           quarter: 201415 }
+    let!(:taken_course5) { create :taken_course, user: user, course: course5,
+                           quarter: 201415 }
+    let!(:taken_course6) { create :taken_course, user: user, course: course6,
+      quarter: 201415 }
+    it "should respond with proper amount" do
+      expect(user.required_credits_earned).to eq 9
+    end
+  end
+
+  describe "#distribution_credits_earned" do
+    before { user.save }
+    let!(:course1) { create :course, degree_requirement: :required_course }
+    let!(:course2) { create :course, degree_requirement: :required_course }
+    let!(:course3) { create :course, degree_requirement: :required_course }
+    let!(:course4) { create :course,
+                     degree_requirement: :distribution_requirement }
+    let!(:course5) { create :course,
+                     degree_requirement: :distribution_requirement }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 113, 68 should be on its own line.